migrations/805-add-addon-is_listed.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE addons
```

Comment:
@jasonthomas? From what I can remember, adding a column with a default to the addons table takes a few minutes, could you please confirm?